lib/cartopy/io/img_tiles.py

```diff
@@ -39,49 +39,14 @@
 import cartopy.crs as ccrs
 
 
-class GoogleTiles(object):
+class GoogleWTS(object):
```

Comment:
Should this derive from `ABCMeta`? Our other base classes seem to do so.

Reply:
ABC is pretty heavweight - personally I avoid it like the plague. Happy to implement if you feel strongly though.

Reply:
**Re: our other base classes**

I can't argue with that. Consistency is definitely a thing. It is pretty galling for me though 😄